bl/evaluation/evaluator_test.go

```diff
@@ -78,11 +79,15 @@ func TestCreateEvaluation(t *testing.T) {
 		cliVersion := "0.0.7"
 		k8sVersion := "1.18.1"
 		policyName := "Default"
+		ciContext := &ciContext.CIContext{
+			IsCI:  true,
+			CIEnv: "travis",
+		}
 
 		mockedCliClient.On("CreateEvaluation", mock.Anything).Return(&cliClient.CreateEvaluationResponse{EvaluationId: 1, K8sVersion: k8sVersion}, nil)
 
 		expectedCreateEvaluationResponse := &cliClient.CreateEvaluationResponse{EvaluationId: 1, K8sVersion: k8sVersion}
-		createEvaluationResponse, _ := evaluator.CreateEvaluation(cliId, cliVersion, k8sVersion, policyName)
+		createEvaluationResponse, _ := evaluator.CreateEvaluation(cliId, cliVersion, k8sVersion, policyName, ciContext)
 
 		mockedCliClient.AssertCalled(t, "CreateEvaluation", mock.Anything)
```

Comment:
We're not testing anywhere that we actually called cli-service with the right params. Would you like to add it at least for the `ciContext`?